triton-vm/src/table/processor_table.rs

```diff
@@ -2851,10 +3119,14 @@ mod constraint_polynomial_tests {
 
     #[test]
     fn instruction_deselector_gives_0_for_all_other_instructions_test() {
-        let deselectors = InstructionDeselectors::default();
+        let mut factory = RowPairConstraints::default();
+        let deselectors = InstructionDeselectors::new(&mut factory);
 
         let mut row = vec![0.into(); 2 * FULL_WIDTH];
 
+        // We need dummy challenges to do partial evaluate. Even though we are
+        // not looking at extension constraints, only base constraints
+        let dummy_challenges = AllChallenges::placeholder();
```

Comment:
How do I convince myself that embedding placeholders will not propagate to some constraint in an accidental way?

Reply:
If we make dummy challenges random every time, then if they pop up accidentally in some constraint, then the prover and verifier will no agree on the constraints. As a result, even honest proofs will fail to validate.